types/catalog/catalog-tests.ts

```diff
@@ -0,0 +1 @@
+import { render } from "catalog";
```

Comment:
I think some test should be added in this file, you may refer to: https://docs.catalog.style/configuration